components/scream/src/physics/shoc/tests/shoc_diag_second_mom_srf_test.cpp

```diff
@@ -115,16 +115,16 @@ struct UnitWrap::UnitTest<D>::TestSecondMomSrf {
       shoc_diag_second_moments_srf_f(d.shcol, d.wthl_sfc, d.uw_sfc, d.vw_sfc, d.ustar2, d.wstar);
     }
 
-#ifndef NDEBUG
-    static constexpr Int num_runs = sizeof(mom_srf_data_f90) / sizeof(SHOCSecondMomentSrfData);
-    for (Int i = 0; i < num_runs; ++i) {
-      Int shcol = mom_srf_data_cxx[i].shcol;
-      for (Int k = 0; k < shcol; ++k) {
-        REQUIRE(mom_srf_data_f90[i].ustar2[k] == mom_srf_data_cxx[i].ustar2[k]);
-        REQUIRE(mom_srf_data_f90[i].wstar[k]  == mom_srf_data_cxx[i].wstar[k]);
+    if (SCREAM_BFB_TESTING) {
```

Comment:
This code is ifdeff-ed out. Is there a reason to keep it? Should we take this chance to purge it? Tagging also @bogensch , which git blame says wrote these lines...

Reply:
Don't care either way.

Reply:
@bargol I'd prefer to keep these lines of code.  Not sure why they were iffdeff-ed in the first place.

Reply:
Should they be compiled/run then?

Reply:
I think they should be?  I guess after looking at it again I'm a little less confident at giving an answer.  While I wrote the property test section of the code for this file, I didn't write the b4b testing section (for any functions) so I'm less familiar.  I'm surprised it said I wrote these lines.  Thus I'm not clear what the issue was with this block of code in the first place to iffdeff them out...  but ustar2 and wstar are the main outputs of this routine, so I would think they should be checked for bit-for-bit ness.

Reply:
Ok. If you feel like this should be kept (and possibly cleaned up/enhanced), can you please go ahead and open an issue, explaining what these lines should do/test. So hopefully we won't forget.